### ISSUE TITLE
New metrics: `installers_new_certifications`, `installers_dropped_certifications`

### ISSUE THREAD

**Issue by mindrones** [nestauk/asf_hp_market_tracker#156]
We need to add 2 metrics in `shared/counts.js`:
- amount of installers getting HP certified
- amount of installers dropping their HP certificate

In `fe/src/lib/statechart/actions/view.js`, for `count/time/` we can use the below but for maps we might need special logic in the backend.

[1]
```js
case 'installers_dropped_certifications':
	aggId = 'date_histogram1_cardinality2';
	params = {
		calendar_interval1: ctx.selection.interval,
		field1: 'installer_certificate_date_end',
		field2: 'installer_id_hash.keyword',
	};
	break;
case 'installers_new_certifications':
	aggId = 'date_histogram1_cardinality2';
	params = {
		calendar_interval1: ctx.selection.interval,
		field1: 'installer_certificate_date_start',
		field2: 'installer_id_hash.keyword',
	};
	break;
```

**Comment by doogyb** [comment]
closed by #163

**Comment by doogyb** [comment]
closed by #163
